### ISSUE TITLE
What is accuracy?

### ISSUE THREAD

**Issue by yakiyaku** [tensorflow/tensorboard#3151]
I opened scalars on tensorboard. Is accuracy the accuracy of the learned data?
I'm not sure
Please give me a professor if you have any details.

**Comment by rmothukuru** [comment]
@yakiyaku,
Accuracy in the Tensorboard can be the Accuracy  on the Training Data and the Validation Data. For code and the procedure to visualize Accuracy mentioned above, please refer the [above link](https://www.tensorflow.org/tensorboard/get_started#using_tensorboard_with_keras_modelfit). Thanks!

**Comment by yakiyaku** [comment]
> @yakiyaku,
> Accuracy in the Tensorboard can be the Accuracy on the Training Data and the Validation Data. For code and the procedure to visualize Accuracy mentioned above, please refer the [above link](https://www.tensorflow.org/tensorboard/get_started#using_tensorboard_with_keras_modelfit). Thanks!

I'll look into it. Thank you very much.

**Comment by rmothukuru** [comment]
@yakiyaku,
Please confirm if I can close this issue. Thanks!

**Comment by rmothukuru** [comment]
Closing the issue as the clarification has been provided. Thanks!
